openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/envoy_folder/vectors/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pandas==1.3.3
+pyarrow=5.0.0
+spacy==3.1.2
```

Comment:
This requirement installs `click==7.1.2` that conflicts with OpenFL Click requirement.

Reply:
You're rigth. Now this part of code is moved from repo as the vectors are pre-calculated. So it seems, if a user wants to calculate vectors, he will have to run it in another environment. (?)